### ISSUE TITLE
parameter param_dict of create_patches doesn't work 

### ISSUE THREAD

**Issue by QiuBingCheng** [amiratag/ACE#19]
if I use the customized parameter like this
`cd.create_patches(param_dict={'n_segments': [5,10,15]})`
it will use the default parameter after the second discovery image
because you use the "pop" function in the  **_return_superpixels**
`param_dict.pop('n_segments', [15, 50, 80])`

**Comment by yingji425** [comment]
Yes. I also find this problem, so I set a param_dict and copy the value as param_dict_c, and put the copy value to  self._return_superpixels(img, method, param_dict_c), so the value will still be the param_dict after the second loop.
Maybe this can help you

**Comment by QiuBingCheng** [comment]
I just replace the `pop` function with `get` function
anyway thank you for your information :)
